warehouse/oidc/views.py

```diff
@@ -238,6 +238,16 @@ def mint_token(
         publisher = oidc_service.find_publisher(claims, pending=False)
         # NOTE: assert to persuade mypy of the correct type here.
         assert isinstance(publisher, OIDCPublisher)
+    except ReusedTokenError:
+        return _invalid(
+            errors=[
+                {
+                    "code": "invalid-reuse-token",
+                    "description": "valid token, but already used",
```

Comment:
A valid but used token is invalid 🙂 

```suggestion
                    "description": "invalid token: already used",
```